Makefile

```
@@ -4,9 +4,11 @@ GO_FILES := $(shell find . -type f -name "*.go" -print)
 .PHONY: run
 
 k6: $(GO_FILES)
-	xk6 build \
+	xk6 build v0.40.0 \
```

```suggestion
	xk6 build \
```
latest versions (from April at this point) of xk6 already build with w/e is in the actual `go.mod`(s) of the extensions - so you can just bump it in `go.mod` - which you already did ;)

---

I guess I should update my `xk6` binary then :D 
Is it ok to still keep it for backwards compatibility?